autogpt_platform/frontend/src/lib/supabase/middleware.ts

```diff
@@ -1,7 +1,12 @@
 import { createServerClient } from "@supabase/ssr";
 import { NextResponse, type NextRequest } from "next/server";
+import { LOCALES } from "@/lib/utils";
 
-export async function updateSession(request: NextRequest) {
+// TODO: Update the protected pages list
+const PROTECTED_PAGES = ["/monitor", "/build"];
+const ADMIN_PAGES = ["/admin"];
```

Comment:
it may be easier to do admin and NOT PROTECTED so that everything by default is protected